packages/flutter_blue_plus_platform_interface/lib/src/bluetooth_msgs.dart

```diff
@@ -94,6 +95,7 @@ class BmScanSettings {
     required this.androidLegacy,
     required this.androidScanMode,
     required this.androidUsesFineLocation,
+    required this.androidCheckLocationServices,
```

Comment:
Could we give this new attribute a sensible default so that it will not be a breaking change to the interface?

```suggestion
    this.androidCheckLocationServices = true,
```

Reply:
@chipweinberger are you okay with this?

Reply:
sure